contributing_to_docs/term_glossary.adoc

```diff
@@ -432,7 +432,9 @@ of the system, ensuring that all containers expected to be running are actually
 running and that other requests such as builds and deployments are serviced.
 New deployments and configurations are created with the REST API, and the state
 of the system can be interrogated through this endpoint as well.  An OpenShift
-control plane comprises the API server, scheduler, and SkyDNS.
+control plane comprises of the API server, scheduler, and SkyDNS.
+
+Note that the OpenShift "control plane" was previously known as "master" and could still be in the code. Where possible, replace "master" with "control plane" in the documentation.
```

Comment:
This only applies to versions 4.9+. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-conscious-language
> If you are cherry picking from main to enterprise-4.8 or earlier, you must manually cherry pick to include the “(also known as master)” phrasing. This is required only if the phrase “control plane” is introduced for the first time in an assembly or module.